### PR TITLE
Fm backport/v1

### PR DESCRIPTION
**Pull request by victorjulien**
FM fixes backport.

### CODE REVIEW

File: src/flow-manager.c [hunk 1]
```diff
@@ -364,6 +377,16 @@ static void FlowManagerHashRowTimeout(FlowManagerTimeoutThread *td, Flow *f, SCT
...
+ * \brief Clear evicted list from Flow Manager.
+ *        All the evicted flows are removed from the Flow bucket and added
+ *        to the temporary Aside Queue.
+ *
+ * \param td FM timeout thread instance
+ * \param f head of the evicted list
+ */
 static void FlowManagerHashRowClearEvictedList(
         FlowManagerTimeoutThread *td, Flow *f, SCTime_t ts, FlowTimeoutCounters *counters)
 {
```
Comment (jufajardini) [inline src/flow-manager.c:392]: Is it worth/ important to add the missing params to the docs, or not worthy waiting for another iteration? (I suppose not 🤔 )

**Review by jufajardini — APPROVED** [review]
Consistent with the original backports. I understand we'll still have to wait on QA results, but from code alone, it looks good.

**Review by jasonish — APPROVED** [review]

**Comment by suricata-qa** [comment]
WARNING:
 

ERROR: QA failed on SURI_TLPR1_alerts_cmp.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 145 | 140 | 96.55% |
| | SURI_TLPR1_stats_chk | | |
| .app_layer.flow.ftp | 33163 | 36871 | 111.18% |
| .app_layer.error.http.parser | 699 | 728 | 104.15% |
| .app_layer.error.ssh.parser | 124 | 128 | 103.23% | 

Pipeline 23765

**Comment by victorjulien** [comment]
Setting to draft to avoid merge before 7.0.8, as this is postponed to 7.0.9.

**Comment by victorjulien** [comment]
Merged in #12302, thanks!

**Pull request closed without merging**